db/migrate/20200323161500_backfill_sent_to_provider_at.rb

```diff
@@ -0,0 +1,21 @@
+class BackfillSentToProviderAt < ActiveRecord::Migration[6.0]
+  def up
+    ApplicationChoice.all.each do |application_choice|
+      state_change = application_choice.audits.find do |audit|
+        audit.audited_changes["status"] == ["application_complete", "awaiting_provider_decision"]
+      end
```

Comment:
This is the first time that we're using the audit log for something like this!

Reply:
But it's definitely why we've added it!